docs/visualize/controls.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[controls]]
+== Controls Visualization
+
+The Controls visualization enables you to add interactive inputs
+to Kibana dashboards.  You can create two types of inputs:
+a dropdown menu and a radio slider.
+
+To start a *Controls* visualization, open the Visualization application
```

Comment:
This seems like a weird place for this. This is the first step for creating a Controls visualization. I think it makes move this under the "Adding Input Controls" page.